omod/src/main/webapp/template/footerFull.jsp

```diff
@@ -44,7 +45,9 @@
 			<span id="buildDate"><openmrs:message code="footer.lastBuild"/>: <%= org.openmrs.web.WebConstants.BUILD_TIMESTAMP %></span>
 			
 			<span id="codeVersion"><openmrs:message code="footer.version"/>: ${openmrsPlatformVersion}</span>
-			
+
+			<c:if test="${not empty footerExtraData}"><span id="extraData">${footerExtraData}</span> </c:if>
```

Comment:
Change this to `<span id="footer.extraData">` and add it [here](https://github.com/openmrs/openmrs-module-legacyui/blob/521d14ff2edc106b997b50964431aab32b9c67c3/omod/src/main/webapp/resources/css/openmrs.css#L198).